src/app/about/page.tsx

```diff
@@ -1,24 +1,28 @@
 export default function Index() {
   return (
     <>
-      <h1 className="my-16 md:mb-12 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
+      <h1 className="my-16 md:mb-6 text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
       Our Mission
 	    </h1>
 	    <div>
-		  Fighting for affordable, accessible, and reliable public transit for the continually growing Dallas area.  
-	    </div>
-	    <h1>
-        About Us
+		    <h2 className="text-3xl mt-12 mb-4 leading-snug leading-tight">
```

Comment:
https://tailwindcss.com/docs/line-height

Leading snug & tight are overwriting the same thing

Reply:
You're right, should be leading-tight everywhere